devicetypes/smartthings/inovelli-dimmer.src/inovelli-dimmer.groovy

```diff
@@ -0,0 +1,606 @@
+			parameterNumber: 18, size: 1, defaultValue: 10,
+			range          : "1..100", disableValue: 0,
+			description    : "The power level change that will result in a new power report being sent (% of previous report). Example of how the values work: 0 = Disabled, 10 = 10% of previous report, 100 = 100% of previous report."
+		],
+		[
+			name           : "Periodic Power + Energy Reports", key: "periodicPower+EnergyReports", type: "range",
+			parameterNumber: 19, size: 2, defaultValue: 3600,
+			range          : "0..32767",
+			description    : "Time period between consecutive power and energy reports being sent (in seconds). Example of how the values work: 0 = 0 seconds, 1 = 1 second, 32767 = 32767 seconds. The timer resets after every report is sent."
+		],
+		[
+			name           : "Dimming Speed (Z-Wave)", key: "dimmingSpeed(Z-Wave)", type: "range",
+			parameterNumber: 2, size: 1, defaultValue: 101,
+			range          : "0..101",
+			description    : "How fast or slow the light turns dim when you adjust the switch remotely (ie: dimming from 10-20%, 80-60%, etc). Example of how the values work: 0 = Instant On, 1 = 1 second, 100 = 100 seconds. Entering the value of 101 = Keeps the switch in sync with Parameter 1."
+		],
+		[
+			name           : "Energy Reports", key: "energyReports", type: "boolRange",
+			parameterNumber: 20, size: 1, defaultValue: 10,
+			range          : "1..100", disableValue: 0,
+			description    : "The energy level change that will result in a new energy report being sent (% of previous report). Example of how the values work: 0 = Disabled, 10 = 10% of previous report, 100 = 100% of previous report."
+		],
+		[
+			name           : "Ramp Rate", key: "rampRate", type: "range",
+			parameterNumber: 3, size: 1, defaultValue: 101,
+			range          : "0..101",
+			description    : "How fast or slow the light turns on when you press the switch 1x to bring from On to Off or Off to On. Example of how the values work: 0 = Instant On, 1 = 1 second, 100 = 100 seconds. Entering the value of 101 = Keeps the switch in sync with Parameter 1."
+		],
+		[
+			name           : "Ramp Rate (Z-Wave)", key: "rampRate(Z-Wave)", type: "range",
+			parameterNumber: 4, size: 1, defaultValue: 101,
+			range          : "0..101",
+			description    : "How fast or slow the light turns on when you bring your switch from On to Off or Off to On remotely. Example of how the values work: 0 = Instant On, 1 = 1 second, 100 = 100 seconds. Entering the value of 101 = Keeps the switch in sync with Parameter 1."
+		],
+		[
+			name           : "Invert Switch", key: "invertSwitch", type: "boolean",
+			parameterNumber: 7, size: 1, defaultValue: 0,
+			optionInactive : 0, inactiveDescription: "Disabled",
+			optionActive   : 1, activeDescription: "Enabled",
+			description    : "Inverts the switch (Tap Down = On, Tap Up = Off). Example of how the values work: 0 = Disabled, 1 = Enabled. This parameter can be set without a HUB from the Configuration Button."
```

Comment:
I think you can just say "Inverts the switch"

Reply:
Done